test/test_suite.sh

```diff
@@ -3,21 +3,23 @@ set -e
 
 cd test
 
-echo "#########################################"
-echo "#########################################"
-echo "####         Start testing! 🤞       ####"
-echo "#########################################"
-echo "#########################################"
-
-docker --version
-
 export SUITE=$1
 
+echo ""
+echo "▶️  Setting-up \"$SUITE\" test suite"
+echo ""
+
 if [ -z "$SUITE" ]; then
-    echo "SUITE is not set"
+    echo "🚨 SUITE is not set"
     exit 1
 fi
 
+# setup log directory, create "last-ran" file and setup log
+rm -Rf "log/$SUITE"
+mkdir -p "log/$SUITE/wikibase"
```

Comment:
```suggestion
mkdir -p "log/$SUITE/wikibase"
mkdir -p "log/$SUITE/client"
```
... should fix running the `repo_client` suite